src/main/java/me/clip/placeholderapi/updatechecker/UpdateChecker.java

```diff
@@ -90,17 +91,22 @@ private boolean spigotIsNewer() {
       return false;
     }
 
-    String plV = toReadable(pluginVersion);
-    String spV = toReadable(spigotVersion);
-    return plV.compareTo(spV) < 0;
+    int[] plV = toReadable(pluginVersion);
+    int[] spV = toReadable(spigotVersion);
+
+    if (plV[0] < spV[0]) {
+      return true;
+    } else if ((plV[1] < spV[1])) {
+      return true;
+    } else return plV[2] < spV[2];
```

Comment:
```suggestion
    } else {
      return plV[2] < spV[2];
    }
```
Just to keep it consistent.